Datahub.Web/Models/SearchResult.cs

```diff
@@ -37,11 +37,11 @@ public class SearchResult
         [Text(Name = "data_type")]
         public string DataType { get; set; }
 
-        [Nested]
-        [PropertyName("datahub_keywords")]
-        public Keyword[] DatahubKeywords { get; set; }
+        // [Nested]
+        // [PropertyName("datahub_keywords")]
+        // public Keyword[] DatahubKeywords { get; set; }
```

Comment:
Would probably just delete these rather than comment out as we are never going back to separate keyword structures

Reply:
Will do